### ISSUE TITLE
Image stylization not working

### ISSUE THREAD

**Issue by Code-With-Samuel** [Tele-AI/TeleStyle#16]
as per your README.md, image stylization can be run through: python telestyleimage_inference.py     
Everytime i try it, i just get this and nothing further:

(.venv) D:\ayush.khanal\TeleStyle>python telestyleimage_inference.py       
Fetching 0 files: 0it [00:00, ?it/s]
Loading models from: [
    "./models\\Qwen/Qwen-Image-Edit-2509\\transformer\\diffusion_pytorch_model-00001-of-00005.safetensors",
    "./models\\Qwen/Qwen-Image-Edit-2509\\transformer\\diffusion_pytorch_model-00002-of-00005.safetensors",
    "./models\\Qwen/Qwen-Image-Edit-2509\\transformer\\diffusion_pytorch_model-00003-of-00005.safetensors",
    "./models\\Qwen/Qwen-Image-Edit-2509\\transformer\\diffusion_pytorch_model-00004-of-00005.safetensors",
    "./models\\Qwen/Qwen-Image-Edit-2509\\transformer\\diffusion_pytorch_model-00005-of-00005.safetensors"
]

(.venv) D:\ayush.khanal\TeleStyle>
===============================================================

FYI, i have specified the content_ref and style_ref in telestyleimage_inference.py as shown below:

if __name__ == "__main__":
    inference_engine = ImageStyleInference()

    prompt = 'Style Transfer the style of Figure 2 to Figure 1, and keep the content and characteristics of Figure 1.'
        
    content_ref='assets/example/0.png'
    style_ref='assets/example/1-1.png'
    
    with torch.no_grad():
        generated_image = inference_engine.inference(prompt, content_ref, style_ref, seed=123, num_inference_steps=4, minedge=1024)

    save_dir=f'./qwen_style_output/'

    os.makedirs(save_dir,exist_ok=True)
    prefix=style_ref.split('/')[-1].split('.')[0]

    generated_image.save(os.path.join(save_dir, f'{prefix}_result.png'))

    print(f"saved to {os.path.join(save_dir, f'{prefix}_result.png')}")
    

========================================================================
Would be grateful if someone helps me and please consider making your README.md a bit more detailed to avoid such minor confusions.

**Comment by ShiwenZhang-teleai** [comment]
hello, it seems you got stuck when loading Qwen-Image-Edit. How much is your GPU memory? Could you run Qwen-Image-Edit normally? Maybe offloading could help.
